### PR TITLE
Adding PlanetMapper

### PR DESCRIPTION
**Pull request by ortk95**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ortk95** [comment]
Failure on `linux_64`:
```
import: 'planetmapper'
Traceback (most recent call last):
  File "/home/conda/staged-recipes/build_artifacts/planetmapper_1685696904282/test_tmp/run_test.py", line 2, in <module>
    import planetmapper
  File "/home/conda/staged-recipes/build_artifacts/planetmapper_1685696904282/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.11/site-packages/planetmapper/__init__.py", line 100, in <module>
    from . import base, data_loader, gui, kernel_downloader, utils
  File "/home/conda/staged-recipes/build_artifacts/planetmapper_1685696904282/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.11/site-packages/planetmapper/gui.py", line 4, in <module>
    import tkinter as tk
  File "/home/conda/staged-recipes/build_artifacts/planetmapper_1685696904282/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.11/tkinter/__init__.py", line 38, in <module>
    import _tkinter # If this fails your Python may not be configured for Tk
    ^^^^^^^^^^^^^^^
ImportError: libX11.so.6: cannot open shared object file: No such file or directory
Tests failed for planetmapper-1.7.2-pyhd8ed1ab_0.conda - moving package to /home/conda/staged-recipes/build_artifacts/broken
WARNING:conda_build.build:Tests failed for planetmapper-1.7.2-pyhd8ed1ab_0.conda - moving package to /home/conda/staged-recipes/build_artifacts/broken
TESTS FAILED: planetmapper-1.7.2-pyhd8ed1ab_0.conda
```

**Comment by ortk95** [comment]
@conda-forge/help-python ready for review!

File: recipes/planetmapper/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - tqdm
+    - pyproj
+    - typing-extensions
+    - xorg-libx11
+
+test:
+  imports:
+    - planetmapper
+  commands:
+    - pip check
```
Comment (martinRenou) [inline recipes/planetmapper/meta.yaml:41]: You could add a test command `planetmapper --help` if that exists, or `planetmapper --version`

File: recipes/planetmapper/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ortk95/planetmapper
+  summary: 'PlanetMapper: A Python module for visualising, navigating and mapping Solar System observations'
+  license: MIT
+  license_file: LICENSE.txt
```
Comment (martinRenou) [inline recipes/planetmapper/meta.yaml:49]:
```suggestion
  license_file: LICENSE.txt
  doc_url: https://planetmapper.readthedocs.io/
  dev_url: https://github.com/ortk95/planetmapper
```

**Review by martinRenou — CHANGES_REQUESTED** [review]
Thanks! I have a couple of suggestions

File: recipes/planetmapper/meta.yaml (hunk 1 above)
Comment (ortk95) [reply recipes/planetmapper/meta.yaml:41]: planetmapper doesn't currently have any commands that don't launch a GUI, but I'll remember to add a test command if/when we add e.g. `planetmapper --help` functionality

**Comment by ortk95** [comment]
Thanks for the suggestions - is there anything else you think needs changing?

**Comment by ortk95** [comment]
Thanks! Is there anything else that I need to do before this can be merged?

**Comment by carterbox** [comment]
This package does not depend on X11 directly, thus it is was not appropriate to add X11 to the dependencies section. Our bots are headless and thus do not have the X11 libs available by default, so I added the correct metadata file to ask the bot to install the display libs as per our documentation.

**Pull request merged**